mlfinlab/portfolio_optimization/mean_variance.py

```diff
@@ -123,9 +123,9 @@ def allocate(self,
 
         # Calculate the portfolio risk and return if it has not been calculated
         if self.portfolio_risk is None:
-            self.portfolio_risk = self.weights @ cov @ self.weights.T
+            self.portfolio_risk = np.dot(self.weights, np.dot(cov.values, self.weights.T))
```

Comment:
Should we have a unit test to assert that this doesn't happen in the future?

Reply:
@Jackal08 The previous code was also fine. I dont know why the error was occuring (Brad raised this error to me). If possible I will add a test to address this in the future.